deploy/charts/istio-csr/values.yaml

```
@@ -205,6 +205,8 @@ app:
       # The type of private key to generate for the serving certificate. Only RSA (default) and ECDSA are supported.
       # NB: This variable is named incorrectly; it controls private key algorithm, not signature algorithm.
       signatureAlgorithm: "RSA"
+    # A list of service accounts that are allowed to use node authentication for CSRs, eg. istio-system/ztunnel
+    caTrustedNodeAccounts: ""
```

nitpick (non-blocking): It might be worth saying that this is a comma-separated list, to make it easier for people to know how to specify the values.

```suggestion
    # A comma-separated list of service accounts that are allowed to use node authentication for CSRs, eg. "istio-system/ztunnel"
    caTrustedNodeAccounts: ""
```